util/make_timeline.py

```diff
@@ -134,9 +168,61 @@ def parse_file(args):
 
     return entries, last_ability_time
 
+def find_fights_in_file(file):
+    e_starts = []
+    encounter_start_staging = 0 # Staged to avoid spurious starts such as limit break usage
+    e_ends = []
+    encounter_in_progress = False
+    current_instance = "none"
+    encounter_sets = []
+    for line in file:
+        # Ignore log lines that aren't game status info
+        if not (line[37:41] == '0839') and not (line[0:2]in ['00','01','33','36']):
```

Comment:
Oof, true, I haven't got pylint (or even pip) set up on this machine since I don't do much Python work. I'll ensure that's taken care of before finalizing the pull request.